.github/workflows/auto-assign-to-project.yml

```diff
@@ -0,0 +1,26 @@
+name: Auto Assign to Project(s)
+
+on:
+  pull_request:
+    types: [opened, labeled]
+env:
+  MY_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I really dislike this but it seems [`srggrs/assign-one-project-github-action@1.2.0`](https://github.com/srggrs/assign-one-project-github-action) requires this... :disappointed: